src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-file-panel.component.ts

```
@@ -2,57 +1,66 @@
-            .first();
+            .first(),
+    );
 
-        this.previousFilePath = this.diffEntries
-            .map((entry) => entry.previousFilePath)
-            .filter((filePath) => filePath)
-            .first();
+    readonly addedLineCount = computed(
+        () =>
+            this.diffEntries()
+                .filter((entry) => entry && entry.filePath && entry.startLine && entry.lineCount)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)